src/osdag/cad/items/filletweld.py

```diff
@@ -82,7 +82,7 @@ def create_model(self,rotate_angle=None):
     from OCC.Display.SimpleGui import init_display
 
     display, start_display, add_menu, add_function_to_menu = init_display()
-    from OCC.gp import gp_Pnt
+    # from OCC.gp import gp_Pnt
```

Comment:
Delete this line if there is no use of `gp_Pnt` anywhere in this file. If there is, fix it similar to how you did elsewhere in this PR.